src/e2e-test/features/servicenowsink/RunTimeWithMacros.feature

```diff
@@ -0,0 +1,198 @@
+    And Enter runtime argument value from environment variable "rest.api.endpoint" for key "restApiEndpoint"
+    And Enter runtime argument value from environment variable "pipeline.user.username" for key "username"
+    And Enter runtime argument value from environment variable "pipeline.user.password" for key "password"
+    And Run the preview of pipeline with runtime arguments
+    Then Verify the preview of pipeline is "success"
+
+  @TS-SN-RNTM-SINK-MACRO-04 @SN_SOURCE_CONFIG @SN_RECEIVING_SLIP_LINE @BQ_SOURCE_UPDATE_RECEIVING_SLIP_LINE @BQ_SINK_CLEANUP
+  Scenario: Verify user should be able to deploy and run a pipeline when plugin is configured for Update operation with macros
+    When Open Datafusion Project to configure pipeline
+    And Select plugin: "BigQuery" from the plugins list as: "Source"
+    And Navigate to the properties page of plugin: "BigQuery"
+    And Fill Reference Name
+    And Configure BigQuery source plugin for Dataset and Table
+    And Validate "BigQuery" plugin properties
+    And Capture the generated Output Schema
+    And Close the Plugin Properties page
+    And Select Sink plugin: "ServiceNow" from the plugins list
+    And Connect plugins: "BigQuery" and "ServiceNow" to establish connection
+    And Navigate to the properties page of plugin: "ServiceNow"
+    And Fill Reference Name
+    And Select radio button plugin property: "operation" with value: "UPDATE"
+    And Click on the Macro button of Property: "tableName" and set the value to: "tableName"
+    And Click on the Macro button of Property: "clientId" and set the value to: "clientId"
+    And Click on the Macro button of Property: "clientSecret" and set the value to: "clientSecret"
+    And Click on the Macro button of Property: "restApiEndpoint" and set the value to: "restApiEndpoint"
+    And Click on the Macro button of Property: "user" and set the value to: "username"
+    And Click on the Macro button of Property: "password" and set the value to: "password"
+    And Validate "ServiceNow" plugin properties
+    And Close the Plugin Properties page
+    And Save and Deploy Pipeline
+    And Run the Pipeline in Runtime
+    And Enter runtime argument value "receiving_slip_line" for key "tableName"
+    And Enter runtime argument value from environment variable "client.id" for key "clientId"
+    And Enter runtime argument value from environment variable "client.secret" for key "clientSecret"
+    And Enter runtime argument value from environment variable "rest.api.endpoint" for key "restApiEndpoint"
+    And Enter runtime argument value from environment variable "pipeline.user.username" for key "username"
+    And Enter runtime argument value from environment variable "pipeline.user.password" for key "password"
+    And Run the Pipeline in Runtime with runtime arguments
+    And Wait till pipeline is in running state
+    And Verify the pipeline status is "Succeeded"
+    Then Open and capture logs
```

Comment:
verify that correct data is written to sink

Reply:
Added the step. (d4426a7b658927fc2fdefd4b5e578521bd150bcf)